google-cloud-speech/README.md

```diff
@@ -24,7 +24,7 @@ $ gem install google-cloud-speech
 ```rb
 require "google/cloud/speech"
 
-speech_client = Google::Cloud::Speech.new
+speech_client = Google::Cloud::Speech::V1p1beta1.new
```

Comment:
Is the intention that the beta API version is the one used in the README? The default version is still showing `:v1` on `Google::Cloud::Speech.new`.